bitcoin/src/blockdata/fee_rate.rs

```diff
@@ -87,6 +87,23 @@ impl FeeRate {
     pub fn checked_mul_by_weight(self, rhs: Weight) -> Option<Amount> {
         self.0.checked_mul(rhs.to_wu()).map(Amount::from_sat)
     }
+
+    /// Calculates the absolute fee by multiplying this fee rate by weight (weight units), returning
+    /// `None` if overflow occurred.
+    ///
+    /// This is equivalent to `Self::checked_mul_by_weight()`.
+    pub fn absolute_fee_wu(self, weight: Weight) -> Option<Amount> {
```

Comment:
Also, why `absolute fee` instead of `fee_wu`?  if the name `checked_mul_by_weight` is confusing, maybe just rename that function instead of adding this wrapper function?

Reply:
Personally I like the `absolute_fee_wu` name, I've always found bdk's `fee_wu` to be quite confusing. I don't have any strong opinion on renaming vs adding a wrapper.

Reply:
What does the work `absolute` mean in the context of fee?  Is there such a thing as a non-absolute fee?

Reply:
I dug around math and english definitions of "absolute" and, contrary to my gut feeling, it does not add any additional meaning although it was also [mis]used in [this Bitocin Core PR](https://github.com/bitcoin/bitcoin/pull/16507/commits/8e59af55aaf1b196575084bce2448af02d97d745).

 I've removed the `absolute_` prefix.